app/views/admin/rebate_forms/_form.html.haml

```diff
@@ -1,8 +1,12 @@
 = form_for [:admin, @rebate_form] do |f|
-  = f.label :valuation_id
-  = f.text_field :valuation_id
-  = f.label :token
-  = f.text_field :token
+  - @rebate_form.fields.each do |item|
```

Comment:
Can we whitelist this? only the fiels that are supposed to be editable.

Reply:
the end user at the council doesn't know about the concept of "fields". To them it's just parts of the application. Can we make it clearer? It's just the 4 fields mentioned in the issue that need to be in a form. 
There's one string, 2 numbers, and one boolean field.

Making the "did you live here on the 1st July" into a free text field would open it up for people to write essays in that, and we don't want that.